### ISSUE TITLE
Android - Object is not a constructor (evaluating 'new _reactNativeBiometrics.default()')

### ISSUE THREAD

**Issue by nppull** [SelfLender/react-native-biometrics#212]
`const rnBiometrics = new ReactNativeBiometrics();`
`TypeError: Object is not a constructor (evaluating 'new _reactNativeBiometrics.default()')`

I get error on android, react native 0.67.0, react-native-biometrics@3.0.1

**Comment by nppull** [comment]
work fine with
react native 0.67.0
react-native-biometrics@2.1.4

`import ReactNativeBiometrics from 'react-native-biometrics';`
```
ReactNativeBiometrics.isSensorAvailable()
      .then((resultObject) => {
        const { available, biometryType } = resultObject;

        console.log('---- zzzz : ' + available + ' ----- ' + biometryType);

        if (available && biometryType === ReactNativeBiometrics.TouchID) {
          console.log('---- TouchID is supported');
        } else if (available && biometryType === ReactNativeBiometrics.FaceID) {
          console.log('---- FaceID is supported');
        } else if (available && biometryType === ReactNativeBiometrics.Biometrics) {
          console.log('---- Biometrics is supported');
        } else {
          console.log('---- Biometrics not supported');
        }
      });
```
```
ReactNativeBiometrics.createKeys()
      .then((resultObject) => {
        const { publicKey } = resultObject;
        console.log('---- public Key: ' + publicKey);
        // sendPublicKeyToServer(publicKey);
      });
```

**Comment by fellipe-ribeiro** [comment]
Same issue here, with both IOS and Android.
React-native 68.2

**Comment by fellipe-ribeiro** [comment]
> work fine with react native 0.67.0 react-native-biometrics@2.1.4
> 
> `import ReactNativeBiometrics from 'react-native-biometrics';`
> 
> ```
> ReactNativeBiometrics.isSensorAvailable()
>       .then((resultObject) => {
>         const { available, biometryType } = resultObject;
> 
>         console.log('---- zzzz : ' + available + ' ----- ' + biometryType);
> 
>         if (available && biometryType === ReactNativeBiometrics.TouchID) {
>           console.log('---- TouchID is supported');
>         } else if (available && biometryType === ReactNativeBiometrics.FaceID) {
>           console.log('---- FaceID is supported');
>         } else if (available && biometryType === ReactNativeBiometrics.Biometrics) {
>           console.log('---- Biometrics is supported');
>         } else {
>           console.log('---- Biometrics not supported');
>         }
>       });
> ```
> 
> ```
> ReactNativeBiometrics.createKeys()
>       .then((resultObject) => {
>         const { publicKey } = resultObject;
>         console.log('---- public Key: ' + publicKey);
>         // sendPublicKeyToServer(publicKey);
>       });
> ```

I think using this old style with the last versions (2.2.2 or 3.0.0) will work too.
Maybe this bug is only in the new versions of React Native with new style of implementation, but still working with the old implementation?

**Comment by jayfunk** [comment]
This might be the same issue as this one https://github.com/SelfLender/react-native-biometrics/issues/214

**Comment by 0mayur1** [comment]
work fine with
react native 0.68.2
react-native-biometrics@2.1.4

i`mport ReactNativeBiometrics from 'react-native-biometrics'`

```
  ReactNativeBiometrics.createKeys()
    .then((resultObject) => {
      const { publicKey } = resultObject
      console.log('/',publicKey)
    })
```

**Comment by fellipe-ribeiro** [comment]
> work fine with react native 0.68.2 react-native-biometrics@2.1.4
> 
> i`mport ReactNativeBiometrics from 'react-native-biometrics'`
> 
> ```
>   ReactNativeBiometrics.createKeys()
>     .then((resultObject) => {
>       const { publicKey } = resultObject
>       console.log('/',publicKey)
>     })
> ```

Yes, work fine in this way even with version 2.2.2, but with constructor don't work in any version

**Comment by fellipe-ribeiro** [comment]
> This might be the same issue as this one #214

Only work without the constructor (old docoumentation)
